src/main/java/bg/jug/website/cms/service/ArticleService.java

```diff
@@ -101,7 +97,7 @@ private Response saveArticleInternal(Article article) {
 	}
```

Comment:
There is one more case here: when the ID is not null, but article with such ID does not exist in the DB. In such cases you have to return status CREATED as well